src/screens/AddScreen.js

```diff
@@ -100,7 +128,7 @@ export default function AddScreen({ navigation }) {
           lat =>
```

Comment:
1. disable editing of latitude when geolocation is "checked", aka when "checked" == true. This is to ensure that if the user is using their geolocation to tag a tree's location, they cannot edit the latitude at the same time.
2. ensure that anything that is pre-existing in the latitude field is cleared and not submitted at the same time as geolocation (this is possible if they started typing and then decided to check geolocation afterwards)

Do the same thing for longitude.